packages/docusaurus-theme-classic/src/theme/CodeBlock/styles.module.css

```diff
@@ -55,3 +55,29 @@
     white-space: pre-wrap;
   }
 }
+
+.codeButton {
+  --docusaurus-code-button-size: 2rem;
+  position: absolute;
+  right: calc(var(--ifm-pre-padding) / 2);
+  top: calc(var(--ifm-pre-padding) / 2);
+  width: var(--docusaurus-code-button-size);
+  height: var(--docusaurus-code-button-size);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: inherit;
+  border: 1px solid var(--ifm-color-emphasis-300);
+  border-radius: var(--ifm-global-radius);
+  transition: opacity 200ms ease-in-out;
+  opacity: 0;
+}
```

Comment:
we should try to separate this CSS in 2 parts:
- the "inner" of a code button (can be extracted and moved to a separate CodeButton component)
- the rules that are responsible to position the button inside the parent component